src/dodal/devices/i10/i10_apple2.py

```diff
@@ -359,14 +360,16 @@ def __init__(
 
         with self.add_children_as_readables():
             super().__init__(
-                id_gap=UndulatorGap(name="id_gap", prefix=prefix),
-                id_phase=UndulatorPhaseAxes(
-                    name="id_phase",
-                    prefix=prefix,
-                    top_outer="RPQ1",
-                    top_inner="RPQ2",
-                    btm_inner="RPQ3",
-                    btm_outer="RPQ4",
+                apple2_motors=Apple2Motors(
+                    id_gap=UndulatorGap(name="id_gap", prefix=prefix),
+                    id_phase=UndulatorPhaseAxes(
+                        name="id_phase",
+                        prefix=prefix,
+                        top_outer="RPQ1",
+                        top_inner="RPQ2",
+                        btm_inner="RPQ3",
+                        btm_outer="RPQ4",
+                    ),
```

Comment:
Are the names necessary here? If you want the naming to always be id_gap and id_phase, why not just call them that in `Apple2Motors` so that the naming is done automatically correctly?

```suggestion
                apple2_motors=Apple2Motors(
                    id_gap=UndulatorGap(prefix),
                    id_phase=UndulatorPhaseAxes(
                        prefix=prefix,
                        top_outer="RPQ1",
                        top_inner="RPQ2",
                        btm_inner="RPQ3",
                        btm_outer="RPQ4",
                    ),
```